api/src/test/java/ca/bc/gov/educ/eas/api/BaseEasAPITest.java

```diff
@@ -55,9 +57,9 @@ public SessionEntity createMockSessionEntity() {
             .courseMonth(Integer.toString(currentDate.getMonthValue()))
             .activeFromDate(currentDate.minusMonths(2))
             .activeUntilDate(currentDate.plusMonths(2))
-            .createUser("ABC")
+            .createUser("EAS_API")
```

Comment:
Please grab this out of the application props

Reply:
Changes applied.